src/lib/locationUtils.js

```diff
@@ -25,6 +25,18 @@ export function isWithinRegion(test, region) {
   )
 }
 
+export function getBounds({start, end}) {
+  if (!start || !end) {return []}
+  let leftX, topY, rightX, bottomY
+  leftX = Math.min(start.row, end.row)
+  topY = Math.max(start.column, end.column)
+  rightX = Math.max(start.row, end.row)
+  bottomY = Math.min(start.column, end.column)
+  return [{row: leftX, column: bottomY}, {row: rightX, column: topY}]
```

Comment:
I think that'd be a good change but I'd want to make that across the suite of location code, not just for the auto fill region.